TombEngine/Scripting/Internal/TEN/Objects/Lara/LaraObject.cpp

```diff
@@ -10,6 +10,7 @@
 #include "Game/effects/item_fx.h"
 #include "Specific/level.h"
 #include "ReservedScriptNames.h"
+#include "Scripting/Internal/TEN/Objects/ObjectsHandler.h"
```

Comment:
Is this needed or a holdover from when you were using GetByName?

Reply:
Yes, I have removed it :)